app/models/request.rb

```diff
@@ -12,17 +12,21 @@ class Request < ApplicationRecord
   validates :ram_mb, numericality: { greater_than: 0, less_than: 257_000 }
   validates :storage_mb, numericality: { greater_than: 0, less_than: 1_000_000 }
 
-  def description_text
-    description = "- Name: #{name}\n"
-    description += "- CPU cores: #{cpu_cores}\n"
-    description += "- RAM: #{ram_mb} MB\n"
-    description += "- Storage: #{storage_mb} MB\n"
-    description += "- Operating System: #{operating_system}"
-    description += comment.empty? ? '' : "\n- Comment: #{comment}"
+  def description_text(host_name)
+    description = "- VM Name: #{name}\n"
+    description += "- Responsible: TBD\n"
+    description += comment.empty? ? '' : "- Comment: #{comment}\n"
+    description += get_url(host_name) + "\n"
     description
   end
 
   def accept!
     self.status = 'accepted'
   end
+
+  private
+
+  def get_url(host_name)
```

Comment:
Is `get_url` a good name? Maybe `url` is sufficient?

Reply:
Thank you for the review.
Fair point, the get_ does not really add value, I changed the name as suggested.